src/resolvers/ArticleResolver.ts

```diff
@@ -2,65 +2,69 @@ import { Resolver, Mutation, Arg, Query, FieldResolver, Root } from 'type-graphq
 import { Article } from '../entities/Article';
 import ArticleRepo from '../repos/ArticleRepo';
 import { DEFAULT_LIMIT } from '../common/constants';
+import UserRepo from '../repos/UserRepo';
 
 @Resolver((_of) => Article)
 class ArticleResolver {
-  @Query((_returns) => Article, { nullable: true })
+  @Query((_returns) => Article, { nullable: true, description: 'Returns a single <Article> via the given <id>' })
```

Comment:
Do these descriptions show up in the Docs on the GraphQL playground?

Reply:
^ bump

Reply:
Yes it does